### ISSUE TITLE
Feature Request: Another marker on invoices. "Paid in full"

### ISSUE THREAD

**Issue by noseshimself** [dragotin/kraft#193]
Could we get a new tag/boolean value on invoices to mark them as paid?

And after that a list of all documents of (the same type) showing creation date, recipient, total and paid/not paid?

**Comment by dragotin** [comment]
No, a bool for payment is not enough. I am actually thinking of a series of states a document can be in, ie. draft, finalized (with an doc id from the series), sent-out (to the customer), paid, rejected, obsoleted etc.

Obviously this kind of little workflow states have to be different per document type (ie. offers are not paid)

My plan is to build that with the XML based docs.

**Comment by noseshimself** [comment]
If someone pays for an offer I'll take the money and run. But see #194.
